pkg/cmd/docs.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+var docsURL string = "https://www.daytona.io/docs/"
+
+var DocsCmd = &cobra.Command{
+	Use:     "docs",
+	Short:   "Opens the Daytona documentation in your default browser and outputs the link to it",
+	Args:    cobra.NoArgs,
+	Aliases: []string{"documentation", "doc"},
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Printf("Opening the Daytona documentation in your default browser. If opening fails, you can go to %s manually.\n", docsURL)
```

Comment:
I suggest we style this a bit better. Take a look at `pkg/views/common.go` and try some "prettier" outputs 😄 

The highlight of the output should be the docs link.